python/tk_multi_workfiles/entity_models/deferred_model.py

```diff
@@ -519,6 +520,23 @@ def _on_deferred_data_refreshed(self, sg_entity, changed, pending_refresh=False)
                 # The item might be already gone because one its ancestor was
                 # deleted.
                 self._delete_item(item)
+        # If we have any uid which is not in both sets, then things changed.
```

Comment:
what if it's in one set but not the other? Does that not mean that things have changed?

Reply:
will clarify the comment: the difference returns the list of uids which are either in one or the other, but not both. Given that `symmetric_difference` is as obscured as `^` to me, I will keep the operator, but have a better comment.